src/WinRT.Interop.Generator/Builders/InteropTypeDefinitionBuilder.Delegate.cs

```diff
@@ -45,7 +45,7 @@ public static void IIDs(
                 interopDefinitions: interopDefinitions,
                 interopReferences: interopReferences,
                 module: module,
-                iid: Guid.NewGuid(), // TODO
+                iid: GuidGenerator.CreateIID(delegateType, interopReferences), // TODO
```

Comment:
Remove the `// TODO` if this line has been replaced with the correct call. Same in the other files.